lib/UA.js

```diff
@@ -665,6 +665,16 @@ module.exports = class UA extends EventEmitter
            * and without To tag.
            */
           break;
+        case JsSIP_C.NOTIFY:
+          /**
+           * receive new sip event
+           */
+          this.emit('sipEvent', {
+            event: request.event.event,
+            body: request.body
```

Comment:
Providing the full request instead of just the body would be desirable. The user can get the body as well as inspect any SIP header, etc.